src/bin/travis/resources/centaur.wdl

```diff
@@ -22,7 +22,7 @@ task centaur {
 
     runtime {
         docker: "geoffjentry/centaur-cromwell:latest" 
```

Comment:
Blurgh.  Would you mind writing a ticket for that?  I'd do it myself but I don't know all the things to ask to be done.  😄